homeassistant/components/yeelight/light.py

```diff
@@ -99,7 +99,9 @@
     'ceiling1': BulbType.WhiteTemp,
     'ceiling2': BulbType.WhiteTemp,
     'ceiling3': BulbType.WhiteTemp,
-    'ceiling4': BulbType.WhiteTempMood}
+    'ceiling4': BulbType.WhiteTempMood,
+    'white_temp': BulbType.WhiteTemp,
+    'white_temp_ambilight': BulbType.WhiteTempMood}
```

Comment:
Do you have a reference where you got these new types? The documentation needs to be updated accordingly: https://www.home-assistant.io/components/yeelight/#model

Reply:
I've started working on this, and did fix on not clean code. I've pushed cleanup